### DISCUSSION TITLE
11.07.2023

### DISCUSSION THREAD

**Discussion by charleslavon** [near/near-discovery#811] [Ideas]
### 🎉 New Features
* Add /relayer-demo route by @Patrick1904 in https://github.com/near/near-discovery/pull/796
* feat: Update terms and privacy by @shelegdmitriy in https://github.com/near/near-discovery/pull/786
* feat: Prepare settings page for idOS integration by @shelegdmitriy in https://github.com/near/near-discovery/pull/728
* Added Ledger to wallet selector by @erditkurteshiSQA in https://github.com/near/near-discovery/pull/800
### 🐛 Fixes
* Fixed navigation menu auto-hide in mobile by @erditkurteshiSQA in https://github.com/near/near-discovery/pull/792
### Enhancements 🌟
* Support auto generated changelogs by @charleslavon in https://github.com/near/near-discovery/pull/789
* NEAR Org Landing Pages (Part 2 of 2) by @calebjacob in https://github.com/near/near-discovery/pull/798

## New Contributors
* @mpeterdev made their first contribution in https://github.com/near/near-discovery/pull/4
* @milenakrawczyk made their first contribution in https://github.com/near/near-discovery/pull/2
* @Patrick1904 made their first contribution in https://github.com/near/near-discovery/pull/7
* @jon-lewis made their first contribution in https://github.com/near/near-discovery/pull/20
* @ChaoticTempest made their first contribution in https://github.com/near/near-discovery/pull/34
* @thisisjoshford made their first contribution in https://github.com/near/near-discovery/pull/66
* @petarvujovic98 made their first contribution in https://github.com/near/near-discovery/pull/79
* @gutsyphilip made their first contribution in https://github.com/near/near-discovery/pull/55
* @BenKurrek made their first contribution in https://github.com/near/near-discovery/pull/107
* @MaximusHaximus made their first contribution in https://github.com/near/near-discovery/pull/162
* @hcho112 made their first contribution in https://github.com/near/near-discovery/pull/199
* @HarisSQA made their first contribution in https://github.com/near/near-discovery/pull/209
* @0xhsy made their first contribution in https://github.com/near/near-discovery/pull/203
* @jakestutzman made their first contribution in https://github.com/near/near-discovery/pull/252
* @roshaans made their first contribution in https://github.com/near/near-discovery/pull/272
* @itegulov made their first contribution in https://github.com/near/near-discovery/pull/250
* @frolvanya made their first contribution in https://github.com/near/near-discovery/pull/302
* @evgenykuzyakov made their first contribution in https://github.com/near/near-discovery/pull/354
* @Ishatt made their first contribution in https://github.com/near/near-discovery/pull/512
* @gabehamilton made their first contribution in https://github.com/near/near-discovery/pull/751
* @caesium55ds made their first contribution in https://github.com/near/near-discovery/pull/744
* @erditkurteshiSQA made their first contribution in https://github.com/near/near-discovery/pull/792

**Full Changelog**: https://github.com/near/near-discovery/commits/latest

<hr /><em>This discussion was created from the release <a href='https://github.com/near/near-discovery/releases/tag/latest'>11.07.2023</a>.</em>

**Comment by charleslavon** [comment]
@thisisjoshford I've created this release & discussion to get feedback on what type of naming convention would be fit releases for releases on the near-discovery and near-discovery-components repo. Since these repos are not APIs or services to be consumed, I don't think it's necessary, and would be overly cumbersome, to use semantic versioning here. Maybe a pattern of using MM.DD.YYYY as a release name would be sufficient for our needs.  Perhaps I'm wrong? 

What say ye?

**Reply by thisisjoshford** [reply]
Sounds good to me. As long as we're organizing the releases like this I'm onboard :) Only suggestion would be to follow YYYY-MM-DD format for better archival organization.
